arch/arm/boot/dts/overlays/README

```diff
@@ -1762,6 +1762,23 @@ Params: oscillator              Clock frequency for the CAN controller (Hz)
 
         interrupt               GPIO for interrupt signal
 
```

Comment:
There should be two blank lines between entries.

Reply:
done